dart/lib/src/protocol/message.dart

```diff
@@ -0,0 +1,38 @@
+class Message {
+  /// The fully formatted message. If missing, Sentry will try to interpolate the message.
+  final String formatted;
+
+  /// The raw message string (uninterpolated).
+  /// example : "My raw message with interpreted strings like %s",
+  final String message;
+
+  /// A list of formatting parameters, preferably strings. Non-strings will be coerced to strings.
+  final List<dynamic> params;
+
+  Message({this.formatted, this.message, this.params});
+
+  Map<String, dynamic> toJson() {
+    return {
+      'formatted': formatted,
+      'message': message,
+      'params': params,
+    };
+  }
+
+  @override
+  String toString() {
+    return 'Message{formatted: $formatted, message: $message, params: $params}';
+  }
+
+  @override
+  bool operator ==(Object other) =>
```

Comment:
We haven't been implementing these jn thr protocol. Is it really worth it?

Reply:
Not really, I thought maybe it could in some tests, but the existing tests test the serialization equality, so I guess it's not necessary.